src/pages/Homepage/homepage.css

```diff
@@ -3,8 +3,11 @@
   flex-direction: column;
   justify-content: space-between;
 }
-.center-items-container {
+.home-page-container {
+  margin: auto;
+  text-align: center;
+  width: 80vw;
```

Comment:
Avoid using `viewport units` as it causes issues when you scale your project.

Reply:
yes.